src/sentry/static/sentry/app/components/sidebar/sidebarItem.jsx

```diff
@@ -69,6 +71,12 @@ class SidebarItem extends React.Component {
 
     const isTop = orientation === 'top';
     const placement = isTop ? 'bottom' : 'right';
+    const LabelHook = HookOrDefault({
+      hookName: 'sidebar:item-label',
+      defaultComponent: ({children}) => {
```

Comment:
formatting nit, not important or worth changing, but you could write this minorly shorter by not using braces for a function body